src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -65,9 +67,22 @@
     <hr/>
     <h2><%= user.getName() %>'s Sent Messages</h2>
 
-    <hr/>
     <div id="chat">
+        <ul>
+            <%
+                List<Message> messages = (List<Message>) request.getAttribute("messages");
+                for (Message message : messages) {
+                    Instant time = message.getCreationTime();
+                    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEE yyyy-MM-dd HH:mm:ss")
```

Comment:
Are you storing these times in the database?

Reply:
Time here is the creation time field of messages.